frontend/src/assets/css/scaffolding.scss

```
@@ -31,3 +31,13 @@ body {
   clip: rect(0 0 0 0);
   clip-path: inset(100%);
 }
+
+.Toastify__toast--error {
+  color: #222222;
```

Let's use color from our design, we already have variables in assets/variables.scss

```suggestion
  color: #222222;
```